src/components/switch/switch.scss

```diff
@@ -18,8 +18,7 @@
 }
 
 label {
-    line-height: normal;
-
+    // @include mdc-theme.mdc-typography(body2);
```

Comment:
Forgot to remove, or purposefully left? 🙂

Reply:
no just thought you needed to check these commented out stuff later :) 

But now when you asked about it, and reading your comments below, I notice that the label text is not at all similar to other labels, for example the checkbox. Font properties are being inherited from Kompendium instead.

I played around with `@use "@material/typography/mdc-typography";` but couldn't really take any advantage of it. Then I pushed a new fix instead.

Reply:
Not sure what's the most optimal way of making label be similar to others, and don't know why it already doesn't. But I think my fix is good enough.

Reply:
Hmm… so you didn't add this line then? I only asked because it looked like this:

![image](https://user-images.githubusercontent.com/154725/126701868-e42bc14f-38e1-4e57-836a-2209471d1515.png)

That's weird… 🤔